xml/installation-installation-gui_installer.xml

```diff
@@ -238,26 +237,147 @@ controller1,192.168.110.3,b2:72:8d:ac:7c:6f,RACK1,HP-DL360-4PORT,CONTROLLER-ROLE
 myserver4,10.2.10.24,00:14:22:01:23:44,AZ1,,,,
 </screen>
  </section>
-<!-- FIXME: procedure needs to be added here when information received from Cloud 
-  Day0 Squad. Instructions at 
-  https://suse-wiki.dyndns.org/display/ETHOS/SUSE+Manager+Certificate+and+Oneview+Certificate.
-Day0 Squad says that it is critical for this to be included. 
  <section xml:id="discover_servers">
   <title>Optional: Importing Certificates for &susemgr; and &oneview;</title>
   <para>
-   If you intend to use &susemgr;, &oneview; to add servers, those certificates
-   must be accessible to the &install_ui;. Use the following steps to import a
-   &susemgr; certificate.
+   If you intend to use &susemgr; or &oneview; to add servers, certificates for
+   those services must be accessible to the &install_ui;.
   </para>
+  <para>
+   Use the following steps to import a &susemgr; certificate.
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Retrieve the <filename>.pem</filename> file from the &susemgr;.
+    </para>
+<screen>
+ curl -k https://<replaceable>SUSE MANAGER IP address:port</replaceable>/pub/RHN-ORG-TRUSTED-SSL-CERT &gt; <replaceable>PEM_NAME</replaceable>.pem
+</screen>
+   </step>
+   <step>
+    <para>
+     Copy the <filename>.pem</filename> file to the proper location on the
+     &lcm;.
+    </para>
+<screen>
+cd /etc/pki/trust/anchors
+/etc/pki/trust/anchors > sudo cp ~/<replaceable>PEM_NAME</replaceable>.pem .
+</screen>
+   </step>
+   <step>
+    <para>
+     Install the certificate.
+    </para>
+<screen>
+sudo update-ca-certificates
+</screen>
+   </step>
+   <step>
+    <para>
+     Add <replaceable>&susemgr; host IP address</replaceable> if
+     <replaceable>&susemgr;.test.domain</replaceable> is not reachable by DNS.
+    </para>
+<screen>
+sudo vi /etc/hosts
+</screen>
+    <para>
+     Add <replaceable>&susemgr; host IP address</replaceable>
+     <replaceable>&susemgr;.test.domain</replaceable>. For example:
+    </para>
+<screen>
+10.10.10.10 &susemgr;.test.domain
+</screen>
+   </step>
+  </procedure>
+  <para>
+   Use the following steps to import an &oneview; certificate.
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Retrieve the <literal>sessionID</literal>.
+    </para>
+<screen>
+curl -k -H "X-Api-Version:500" -H "Content-Type: application/json" \
+-d '{"userName":<replaceable>ONEVIEW USERNAME</replaceable>, "password":<replaceable>ONEVIEWPASSWORD</replaceable>, \
+"loginMsgAck":"true"}' https://<replaceable>ONEVIEW_MANAGER_URL:PORT</replaceable>/rest/login-sessions
+</screen>
+    <para>
+     The response will be similar to:
+    </para>
+<screen>
+{"partnerData":{},"sessionID":"LTYxNjA1O1NjkxMHcI1b2ypaGPscErUOHrl7At3-odHPmR"}
+</screen>
+   </step>
+   <step>
+    <para>
+     Retrieve a Certificate Signing Request (CSR) using the
+     <replaceable>sessionID</replaceable> from the previous step.
```

Comment:
nit: It generally makes sense to use an xref for these things -- e.g. someone might add a step in between and suddenly the previous step is not the same anymore. It's also easier for readers if they can click on the step name.
i.e. add an ID to the previous step, such as `id="st.oneview.retrieve-id"`, and reference like: `...using the sessionID from <xref linkend="st.oneview.retrieve-id"/>.`